### ISSUE TITLE
Feature request: add repo link to update notifer

### ISSUE THREAD

**Issue by shrpne** [raineorshine/npm-check-updates#1193]
It would be useful to add repository or changelog link to the notification generated by `update-notifier`

It can be done with custom `message` option as described here https://github.com/yeoman/update-notifier/issues/213#issuecomment-1227834475

**Comment by raineorshine** [comment]
Hi, thanks for the suggestion. 

Given that I only add change log notes for major version releases, I think I would prefer without the link. If there was a way to provide a direct link to the release notes only when they exist, I would be open to that.

**Comment by shrpne** [comment]
May be just the link for the repository then? So user can investigate what is changed

**Comment by raineorshine** [comment]
I have given this further consideration, and figured out a way to provide useful information for both major and non-major upgrades without having to change my changelog policy.

Major updates link to the release notes:

```sh
╭────────────────────────────────────────────────────────────────────────────╮
│                                                                            │
│                      Update available 15.3.4 → 16.0.0                      │
│                  Run npm i -g npm-check-updates to update                  │
│   https://github.com/raineorshine/npm-check-updates/releases/tag/v16.0.0   │
│                                                                            │
╰────────────────────────────────────────────────────────────────────────────╯
```

Major updates across multiple major versions link to each major version's release notes:

```sh
╭────────────────────────────────────────────────────────────────────────────╮
│                                                                            │
│                      Update available 13.0.0 → 16.3.1                      │
│                  Run npm i -g npm-check-updates to update                  │
│   https://github.com/raineorshine/npm-check-updates/releases/tag/v14.0.0   │
│   https://github.com/raineorshine/npm-check-updates/releases/tag/v15.0.0   │
│   https://github.com/raineorshine/npm-check-updates/releases/tag/v16.0.0   │
│                                                                            │
╰────────────────────────────────────────────────────────────────────────────╯
```

Non-major updates link to a list of commits between current and latest:

```sh
╭─────────────────────────────────────────────────────────────────────────────────╮
│                                                                                 │
│                        Update available 16.2.0 → 16.3.1                         │
│                    Run npm i -g npm-check-updates to update                     │
│   https://github.com/raineorshine/npm-check-updates/compare/v16.2.0...v16.3.1   │
│                                                                                 │
╰─────────────────────────────────────────────────────────────────────────────────╯
```

Published in `v16.3.2`

**Comment by shrpne** [comment]
Cool! Thank you!
